PublicMapping/districtbuilder
# Add keyboard shortcuts button

## Overview

- Renames the Support menu to Resources
- Adds a button to the Resources menu to Show Keyboard Shortcuts


### Checklist


### Demo

Optional. Screenshots, examples, etc.

### Notes

- ~Pending a keyboard icon from @jfrankl~
- @tgilcrest This now creates a link on the homepage header which is broken - since the Keyboard Shortcuts modal is only accessible thru the Map component. Does it make sense to hide this new button unless the user is within a project? 


## Testing Instructions

- Open a map; click the Resources dropdown and 'Show keyboard shortcuts'
- Expect keyboard shortcuts modal to be displayed

Closes #757
